usr/local/www/diag_dns.php

```diff
@@ -170,116 +171,150 @@ function display_host_results ($address,$hostname,$dns_speeds) {
 }
 
 include("head.inc"); ?>
-<body link="#0000CC" vlink="#0000CC" alink="#0000CC">
-<?php include("fbegin.inc"); ?>
-<table width="100%" border="0" cellpadding="0" cellspacing="0" summary="diag dns">
-        <tr>
-                <td>
-<?php if ($input_errors) print_input_errors($input_errors); ?>
-	<form action="diag_dns.php" method="post" name="iform" id="iform">
-	  <table width="100%" border="0" cellpadding="6" cellspacing="0" summary="tabcont">
-		<tr>
-			<td colspan="2" valign="top" class="listtopic"> <?=gettext("Resolve DNS hostname or IP");?></td>
-		</tr>
-        <tr>
-		  <td width="22%" valign="top" class="vncellreq"><?=gettext("Hostname or IP");?></td>
-		  <td width="78%" class="vtable">
-            <?=$mandfldhtml;?>
-			<table summary="results">
-				<tr><td valign="top">
-			<input name="host" type="text" class="formfld" id="host" size="20" value="<?=htmlspecialchars($host);?>" />
-			</td>
-			<?php if ($resolved && $type) { ?>
-			<td valign="middle">&nbsp;=&nbsp;</td><td>
-			<font size="+1">
+
 <?php
-				$found = 0;
-				if(is_array($resolved)) { 
-					foreach($resolved as $hostitem) {
-						if($hostitem <> "") {
-							echo $hostitem . "<br />";
-							$found++;
+	/* Dosplay any error messages resulting from user input */ 
+	if ($input_errors) 
+		print_input_errors($input_errors);
+	else if (!$resolved && $type)
+		{
+		$errors[] = gettext("Host") . ": " . "\"" . $host . "\"" . " " . gettext("could not be resolved");
+		print_errors($errors, gettext("DNS Lookup error:"));		
+	}
+?>
+  
+<form action="diag_dns.php" method="post" name="iform" id="iform">
+
+<!-- First table displayes the DNS lookup form and the resulting IP addresses -->
+	<div class="table-responsive">
+	
+		<table class="table" summary="results">
+		<thead>
+			<tr>
+				<th width="30%">Host name</th>
+				<?php if ($resolved && $type) { ?> 
+				<th>IP Addresse(s)</th>
+				<?php } ?>
+			</tr>
+		</thead>
+		<tbody>
+			<tr class="active">
```

Comment:
"table table-striped" makes rows alternate dark/light.

“active" overrides that and sets the particular row to the darker shade. So if you choose not to specify "table-striped" it allows you to control which rows have the darker shading.

Steve
--
Stephen Beaver | +1 (512) 646-4100;239 | sbeaver@netgate.com | www.netgate.com 

> On Mar 28, 2015, at 10:56 AM, SjonHortensius <notifications@github.com> wrote:
> 
> In usr/local/www/diag_dns.php <https://github.com/SjonHortensius/pfsense/pull/42#discussion_r27345597>:
> 
> > +<form action="diag_dns.php" method="post" name="iform" id="iform">
> > +
> > +<!-- First table displayes the DNS lookup form and the resulting IP addresses -->
> > +	<div class="table-responsive">
> > +	
> > +		<table class="table" summary="results">
> > +		<thead>
> > +			<tr>
> > +				<th width="30%">Host name</th>
> > +				<?php if ($resolved && $type) { ?> 
> > +				<th>IP Addresse(s)</th>
> > +				<?php } ?>
> > +			</tr>
> > +		</thead>
> > +		<tbody>
> > +			<tr class="active">
> Pretty sure tr.active doesn't match any css; what should it do?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/SjonHortensius/pfsense/pull/42/files#r27345597>.
>